cmd/gateway-azure.go

```diff
@@ -579,14 +635,16 @@ func (a *azureObjects) PutObjectPart(bucket, object, uploadID string, partID int
 		}
 
 		id := azureGetBlockID(partID, subPartNumber, uploadID, etag)
-		err = a.client.PutBlockWithLength(bucket, object, id, uint64(subPartSize), io.LimitReader(data, subPartSize), nil)
+		blob := a.client.GetContainerReference(bucket).GetBlobReference(object)
+		err = blob.PutBlockWithLength(id, uint64(subPartSize), io.LimitReader(data, subPartSize), nil)
 		if err != nil {
 			return info, azureToObjectError(traceError(err), bucket, object)
 		}
 		subPartNumber++
 	}
 	if err = data.Verify(); err != nil {
-		a.client.DeleteBlob(bucket, object, nil)
+		blob := a.client.GetContainerReference(bucket).GetBlobReference(object)
+		_ = blob.Delete(nil)
```

Comment:
As the error from `blob.Delete()` is not returned, we would need to log the error

Reply:
Done.